### PR TITLE
feat(onyx-1240): add Your Active Bids home view section

### PR DESCRIPTION
**Pull request by nickskalkin**
Request:

```graphql
query {
  homeView {
    section(id: "home-view-section-active-bids") {
      ... on ArtworksRailHomeViewSection {
        component {
          title
        }

        artworksConnection(first: 1) {
          edges {
            node {
              title
            }
          }
        }
      }
    }
  }
}
```

Response:

```json
{
  "data": {
    "homeView": {
      "section": {
        "component": {
          "title": "Your Active Bids"
        },
        "artworksConnection": {
          "edges": [
            {
              "node": {
                "title": "Untitled (Tiles)"
              }
            }
          ]
        }
      }
    }
  }
}
```

### CODE REVIEW

File: src/schema/v2/homeView/artworkResolvers.ts [hunk 1]
```diff
@@ -118,3 +119,19 @@ export const AuctionLotsForYouResolver: GraphQLFieldResolver<
...
+
+export const ActiveBidsResolver: GraphQLFieldResolver<
+  any,
+  ResolverContext
+> = async (_parent, args, context, _info) => {
+  const { lotStandingLoader } = context
+
+  if (!lotStandingLoader) return []
+
+  let result = await lotStandingLoader({
```
[the remark this answers is not in the document]
Comment (MounirDhahri) [reply src/schema/v2/homeView/artworkResolvers.ts:131]: The existing implementation backed by the home active lots resolver in the existing home view isn't working for me and I am not getting the active lots. I am only getting them when visiting the inbox tab active bids which is backed by `me.activeBids` [here](https://github.com/artsy/eigen/blob/a0355c3c4d2732964454549a7878a20706ba3f83/src/app/Scenes/MyBids/MyBids.tsx#L183). Maybe it's worth it to use it here as well.

**Review by MounirDhahri — COMMENTED** [review]
LGTM - just a small comment

File: src/schema/v2/homeView/artworkResolvers.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (nickskalkin) [reply src/schema/v2/homeView/artworkResolvers.ts:131]: interesting, @MounirDhahri so this query returns nothing for you even if you have active bids?

```
query {
  homePage {
      artworkModule(key: ACTIVE_BIDS) {
        results {
          title
        }
      }
    }
}
```

[the remark this answers is not in the document]
Comment (MounirDhahri) [reply src/schema/v2/homeView/artworkResolvers.ts:131]: yes, this one doesn't work for me :/ Glad to continue with it if it works for you though and look into it later. `me.activeBid` is a newer interface so there might be something that we forgot to adjust that breaks it for me

**Review by MounirDhahri — APPROVED** [review]

File: src/schema/v2/homeView/artworkResolvers.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (nickskalkin) [reply src/schema/v2/homeView/artworkResolvers.ts:131]: From one hand I think it is at least worth looking at, but from another - if our goal is to achieve parity with current home screen - maybe we should stick to the same resolver here?

[the remark this answers is not in the document]
Comment (damassi) [reply src/schema/v2/homeView/artworkResolvers.ts:131]: Worth noting that `me.activeBids` is particularly slow! We'd need to defer that for sure if it ends up being used.

**Review by anandaroop — APPROVED** [review]
👍🏽

File: src/schema/v2/homeView/artworkResolvers.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (MounirDhahri) [reply src/schema/v2/homeView/artworkResolvers.ts:131]: Thanks for the notice. Let's stick with the old resolver then and we can fix it later

**Pull request merged**